CHANGES

```diff
@@ -22,12 +22,16 @@ description indicates whether it is SOURCE-COMPATIBLE, BINARY-COMPATIBLE, or BRE
 Further information about [seL4 releases](https://docs.sel4.systems/sel4_release/) is available.
 
 ---
-Upcoming release: BINARY COMPATIBLE
+Upcoming release: BREAKING
```

Comment:
I don't think that this is a breaking change.

Reply:
I will test this again, but locally I had an issue that on RISC-V an old elf loader and an new kernel don' fit together, this started all the cleanup activity here.

Reply:
I've remove this, as it is not breaking on the upper layer. It may only affect custom bootloaders.